client/src/components/Topic/TopicCard.jsx

```diff
@@ -11,15 +18,15 @@ export default function BlogPostCard({ topic }) {
         {topic.posts}
       </p>
     </div>
-    <Link to="/topic/:id" className="inline-flex items-center px-3 py-2 text-sm font-medium text-center text-white bg-blue-700 rounded-lg hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">
+    <Link to="/topics/:id" className="inline-flex items-center px-3 py-2 text-sm font-medium text-center text-white bg-blue-700 rounded-lg hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">
```

Comment:
shouldn't be {`/topics/${id}`} or {`/topics/${topic.id}`}?

Reply:
yes :) thanks !